golangci.yaml

```diff
@@ -1,12 +1,35 @@
-linters:
-  disable:
-    - megacheck
+liners:
+  disable-all: true
   enable:
+    - bodyclose
+    - deadcode
+    - depguard
+    - dogsled
+    - errcheck
+    - gocyclo
+    - gofmt
+    - goimports
+    - goprintffuncname
+    - gosec
     - gosimple
+    - govet
+    - ineffassign
+    - interfacer
+    - nakedret
+    - nolintlint
+    - rowserrcheck
+    - scopelint
```

Comment:
`scopelint` is superseded by `exportloopref`.

Reply:
+1

Reply:
exportloopref isn't supported by golangci-lint 1.27 and I am having trouble upgrading it. Can we put this in with scoprelint and upgrade/replace in a subsequent merge?

Reply:
I am ok with doing this later.